pkg/minikube/cluster/cluster.go

```diff
@@ -116,15 +117,24 @@ func StartHost(api libmachine.API, config cfg.MachineConfig) (*host.Host, error)
 		return nil, errors.Wrap(err, "Error getting state for host")
 	}
 
-	if s == state.Running {
-		console.OutStyle(console.Running, "Re-using the currently running %s VM for %q ...", h.Driver.DriverName(), cfg.GetMachineName())
-	} else {
-		console.OutStyle(console.Restarting, "Restarting existing %s VM for %q ...", h.Driver.DriverName(), cfg.GetMachineName())
-		if err := h.Driver.Start(); err != nil {
-			return nil, errors.Wrap(err, "start")
+	if config.VMDriver != constants.DriverNone {
+		if s == state.Running {
+			console.OutStyle(console.Running, "Re-using the currently running %s VM for %q ...", h.Driver.DriverName(), cfg.GetMachineName())
+		} else {
+			console.OutStyle(console.Restarting, "Restarting existing %s VM for %q ...", h.Driver.DriverName(), cfg.GetMachineName())
+			if err := h.Driver.Start(); err != nil {
+				return nil, errors.Wrap(err, "start")
+			}
+			if err := api.Save(h); err != nil {
+				return nil, errors.Wrap(err, "save")
+			}
+
 		}
-		if err := api.Save(h); err != nil {
-			return nil, errors.Wrap(err, "save")
+	} else {
+		if s == state.Running {
+			console.OutStyle(console.Running, "Re-using the currently running %s VM for %q ...", h.Driver.DriverName(), cfg.GetMachineName())
```

Comment:
Should this say VM?

Reply:
Most people should be running `none` on a dedicated VM, but yeah - we normally use "VM" as a term for the host, in the case of "none" or "generic" or "docker" that might not actually be true.

Normally it's called a machine, as in "something runs docker".